dockerfiles/Dockerfile_slim

```diff
@@ -5,8 +5,8 @@ ARG DOWNLOAD_URL=https://developer.salesforce.com/media/salesforce-cli/sfdx/chan
 RUN apt-get update
 # install cli from linux tarball with bundled node
 RUN curl -s $DOWNLOAD_URL --output sfdx-linux-x64.tar.xz \
-  && mkdir ~/sfdx \
-  && tar xJf sfdx-linux-x64.tar.xz -C ~/sfdx --strip-components 1 \
+  && mkdir /usr/local/sfdx \
```

Comment:
it might be hard to verify this locally without hosing your /usr folder, but should this be mkdir -p to make sure all the folders in the chain get created?

Reply:
Ah yes, since `/usr/local` already exists in the container I didn't see the need to add that flag but I like it being idempotent 👍🏼.